src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+date: 2020-02-15
+---
+
+### Wat is Phaser?
+Kort gezegd is [Phaser](https://phaser.io/) een gameframework waarmee je HTML-5 games kan maken. Deze games kunnen gespeeld worden op desktop en mobiel. Phaser kan gratis gedownload worden via https://phaser.io/ en is ontwikkeld door Photon Storm in 2013.
+
+Om gebruik te maken van Phaser is het handig als je al enige kennis hebt van HTML en JavaScript. Phaser is namelijk een JavaScript bibliotheek, wat dus betekent dat je de game volledig hierin bouwt. Phaser maakt het mogelijk voor beginners en gevorderden om een eigen interactieve en leuke game te maken. Verder heeft het veel verschillende functies, zelf te veel om allemaal te benoemen. Dit is ook een van de redenen waarom het zo geliefd is bij game ontwikkelaars. 
+
+### Opzetten van een Phaser game
+Het begint allemaal bij je HTML bestand waar je Phaser oproept in je head, met de volgende code: 
+```
+<script src="https://cdnjs.cloudflare.com/ajax/libs/phaser-ce/2.12.0/phaser.min.js"></script>
+```
+Na dit gedaan te hebben maak je een JavaScript bestand aan en roep je de game op met:
+```
+window.onload = function(){
+    var game = new Phaser.Game(800 (width), 600 (height), Phaser.AUTO, "", {
+        preload: preload,
+        create: create,
+        update: update
+  }); 
+```
+Als weergavecontext voor de game kan er gekozen worden voor Phaser.CANVAS, Phaser.WEBGL of Phaser.AUTO. De aanbevolen waar om te gebruiken is Phaser.AUTO, omdat deze automatisch dat gebruikt wat de browser die wordt gebruikt ondersteunt. De width en de height staan voor de canvasgrootte, in dit geval is dat 800 x 600 pixels. 
```

Comment:
_De aanbevolen waar om_ -> waarde

_omdat deze automatisch dat gebruikt wat de browser die wordt gebruikt ondersteunt._ -> Deze zin is een beetje onduidelijk. 

En ik zou misschien wat extra context verwachten over CANVAS, WEBGL en AUTO. Wat is het verschil?